jaxrs/src/main/java/org/jboss/as/jaxrs/deployment/JaxrsIntegrationProcessor.java

```diff
@@ -559,7 +559,7 @@ private boolean isTransmittable(AttributeDefinition attribute, ModelNode modelNo
         if ("".equals(value.trim())) {
             return false;
         }
-        return !value.equals(attribute.getDefaultValue());
+        return !value.equals(attribute.getDefaultValue() != null ? attribute.getDefaultValue().asString() : null);
```

Comment:
This could just be:
```
return attribute.getDefaultValue() != null && !value.equals(attribute.getDefaultValue().asString())
```

That's a bit of a nit though I guess :)

Reply:
Yes, after reading the smarter solution I wonder why I did the first one. Still a long way for me to go :-)

Reply:
@jamezp I wanted provide the fix but let's check twice:

```
Scenario "attribute.getDefaultValue() returns null"

(1) Current fix of the PR
!value.equals(attribute.getDefaultValue() != null ? attribute.getDefaultValue().asString() : null);
!value.equals(null != null ? attribute.getDefaultValue().asString() : null);
!value.equals(null);
!false;
return true;

(2) Short edition
attribute.getDefaultValue() != null && !value.equals(attribute.getDefaultValue().asString())
null != null && !value.equals(attribute.getDefaultValue().asString())
false && !value.equals(attribute.getDefaultValue().asString())
false && not evaluated anymore
return false;
```

Reply:
Sorry you're right. It should be:
```
return attribute.getDefaultValue() == null || !value.equals(attribute.getDefaultValue().asString())
```

However, that's just a minor nit and really shouldn't hold this PR up.